controllers/lvmcluster_controller.go

```diff
@@ -348,3 +338,47 @@ func (r *LVMClusterReconciler) getRunningPodImage(ctx context.Context) error {
 
 	return nil
 }
+
+func (r *LVMClusterReconciler) logicalVolumesExist(ctx context.Context, instance *lvmv1alpha1.LVMCluster) (bool, error) {
+
+	logicalVolumeList := &topolvmv1.LogicalVolumeList{}
+
+	if err := r.Client.List(ctx, logicalVolumeList); err != nil {
```

Comment:
```suggestion
	if err := r.Client.List(ctx, logicalVolumeList, &client.ListOptions{Limit: int64(1)}); err != nil {
```

Reply:
What is the advantage of this?

Reply:
- we are checking the existence of logicalvolume crs so why not limit the request to list only one item

Reply:
Does it make any difference from a performance point of view?

Reply:
- leaving this then, even though we might be reading from cache I think limiting entries doesn't harm anything for the operation that we are doing in this func